mailer.js

```diff
@@ -7,6 +7,12 @@ var P = require('bluebird')
 var nodemailer = require('nodemailer')
 
 module.exports = function (log) {
+  // Email template to UTM campaign map
+  var utmCampaignMap = {
```

Comment:
I wonder if the intent would be clearer if this were named `templateNameToCampaignMap` - it's verbose, but which item is the key and which is the value is clear.